config-store-yaml/pom.xml

```diff
@@ -41,7 +35,7 @@
       <dependency>
         <groupId>com.fasterxml.jackson.dataformat</groupId>
         <artifactId>jackson-dataformat-yaml</artifactId>
-        <version>2.11.3</version>
+        <version>2.12.5</version>
```

Comment:
In near future, I'll probably try to add `dependencyManagement` for Jackson (similar to Guava here); challenges are similar as Stargate can use newer/-est versions just fine but C*3.11/4.0/DSE depend on older versions (although I am working on trying to get Jackson update for Cassandra too, concurrently).